pom.xml

```diff
@@ -124,6 +124,11 @@
         <artifactId>plexus-utils</artifactId>
         <version>${plexus-utils.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.maven.shared</groupId>
+        <artifactId>maven-common-artifact-filters</artifactId>
+        <version>3.1.0</version>
```

Comment:
Could you please use a variable here, just for consistency with other versions in the POM ?